src/test/scala/fs2/kafka/KafkaConsumerSpec.scala

```diff
@@ -155,5 +175,49 @@ final class KafkaConsumerSpec extends BaseKafkaSpec {
         } yield ()).compile.drain.unsafeRunSync
       }
     }
+
+    def seekTest(numMessages: Long, readOffset: Long, partition: Option[Int] = None)(config: EmbeddedKafkaConfig, topic: String) = {
+      createCustomTopic(topic)
+
+      val produced = (0l until numMessages).map(n => s"key-$n" -> s"value->$n")
+      publishToKafka(topic, produced)
+
+      val consumed =
+          consumerSettings(config).flatMap { consumerSettings =>
+            consumerStream[IO].using(consumerSettings).flatMap { consumer =>
+
+              val validSeekParams =
```

Comment:
This test setup is a bit ugly. Since `KafkaConsumer` does not expose e.g. `partitionsFor`, the only way I can get hold of a valid `TopicPartition`/`offset` pair is to commit a message and look inside the `CommitableOffset`.

Reply:
We could always add `partitionsFor`. But I'm also happy to leave it as is for now and rewrite later.